apps/trading/e2e/fixtures/market.py

```diff
@@ -129,12 +137,19 @@ def setup_opening_auction_market(vega: VegaService, market_id: str = None, **kwa
 
     return market_id
 
+def market_exists(vega: VegaService, market_id: str):
+    if market_id is None:
+        return False
+    all_markets = vega.all_markets() 
+    market_ids = [market.id for market in all_markets]
+    return market_id in market_ids
 
-def setup_continuous_market(vega: VegaService, market_id: str = None, **kwargs):
-    if market_id is None or market_id not in vega.all_markets():
-        market_id = setup_opening_auction_market(vega, **kwargs)
 
-    submit_order(vega, "Key 1", market_id, "SIDE_BUY", 1, 110)
+def setup_continuous_market(vega: VegaService, market_id: str = None, buy_orders = default_buy_orders, sell_orders = default_sell_orders, add_liquidity = True, **kwargs): #Add sell orders and buy orders to put on the book
```

Comment:
not a fan of all of these params but in a separate PR I would like to refactor this so that we can pass market objects around to these methods to make this nicer to configure.

Reply:
Sounds sensible, just converting to a dictionary would be a good start. Fire up an additional PR if you have the capacity

Reply:
sweet.